kfdef/kfctl_openshift.yaml

```diff
@@ -120,4 +127,6 @@ spec:
     uri: https://github.com/kubeflow/manifests/archive/v1.2-branch.tar.gz
   - name: manifests
     uri: https://github.com/opendatahub-io/odh-manifests/tarball/master
+  - name: kf-jupyterhub-patch
```

Comment:
@crobby @anishasthana 
we will have to add this patch source to this file as well right?
https://github.com/red-hat-data-services/odh-deployer/blob/main/opendatahub.yaml